app/mailers/assignment_mailer.rb

```diff
@@ -10,4 +10,14 @@ def update_assignment_email(user,  assignment)
     @user = user
     mail(to: (@user.email), subject: "Assignment Updated in " +  Group.find_by(id:(@assignment.group_id)).name )
   end
+
+  def deadline_assignment_email(assignment, projects_submitted, projects_not_submitted)
+    @assignment = assignment
+    @projects_submitted = projects_submitted
+    @projects_not_submitted = projects_not_submitted
+    @user = User.find_by(id: Group.find_by(id: @assignment.group_id).mentor_id)
```

Comment:
This should be something like

@user = @assignment.group.mentor

Reply:
Thank you!